pythonik/client.py

```diff
@@ -35,6 +32,9 @@ def __init__(self, app_id: str, auth_token: str, timeout: int):
         }
         self.timeout = timeout
 
+    def collections(self):
+        return CollectionSpec(self.session, self.timeout)
```

Comment:
@nsa-prince nice work! Collection as its own seperate class is cleaner. I  think we should to this for other sub specs lets call them that are lengthy such as FileSets which are still part of the File spec. That being said, do you think it still be a good idea to add this spec to the AssetSpec since its part of the Asset spec in the iconik docs -> https://app.iconik.io/docs/apidocs.html?url=/docs/assets/spec/#/

for example

client.assets().collections.get(...)

or maybe as a property or both :) 

client.assets.collections().get(...) for example

Reply:
Implemented as a property!